index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Microverse/freeCodeCamp Survey Form</title>
+</head>
+
+<body>
+  
```

Comment:
Hi [@MostafaShahat2021](https://github.com/MostafaShahat2021), 👋🏻👋🏻

Good job so far! 🔥🔥💯💯🚀🚀
There are some issues that you still need to work on to go to the next project but you are almost there!
Highlights ✨✨

📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the functionality ✅
📌 Good commit messages ✅
📌Professional README file. ✅